geopy/geocoders/osm.py

```diff
@@ -263,6 +264,9 @@ def reverse(
 
             .. versionadded:: 1.0.0
 
+        :param bool addressdetails: Whether or not to include address details,
+        such as city, county, state, etc. in *Location.raw*
```

Comment:
Could you please indent this line? In a similar way as above.

Also it'd be great if you could add the `.. versionadded:: 1.14.0` line below.